posthog/hogql/database/database.py

```diff
@@ -174,7 +174,7 @@ class Database(BaseModel):
         RawErrorTrackingIssueFingerprintOverridesTable()
     )
     raw_sessions: Union[RawSessionsTableV1, RawSessionsTableV2] = RawSessionsTableV1()
-    raw_query_log: RawQueryLogTable = RawQueryLogTable()
+    raw_query_log: RawQueryLogArchiveTable = RawQueryLogArchiveTable()
```

Comment:
I think we want this removed - otherwise people will be able to query _all teams_ query logs

Reply:
I've tried to deal without the raw_query_log, but could not get it to create a proper query.
Should i try harder?